dev/WindowsAppSDK_Insights/packages.config

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Microsoft.Telemetry.Inbox.Native" version="10.0.19041.1-191206-1406.vb-release.x86fre" targetFramework="native" />
```

Comment:
>x86fre [](http://example.com/codeflow?start=94&length=6)

i'd assume this is ok assuming it's header only? but want to get confirmation.

Reply:
That is correct. This is the latest package that is published.